tasks/utilities/task_broad_terra_tools.wdl

```diff
@@ -609,7 +617,15 @@ task export_taxon_tables {
       "agrvate_agr_multiple": "~{agrvate_agr_multiple}",
       "agrvate_agr_num_frameshifts": "~{agrvate_agr_num_frameshifts}",
       "agrvate_version": "~{agrvate_version}",
-      "agrvate_docker": "~{agrvate_docker}"
+      "agrvate_docker": "~{agrvate_docker}",
+      "srst2_vibrio_tsv": "~{srst2_vibrio_tsv}",
+      "srst2_vibrio_detailed_tsv": "~{srst2_vibrio_detailed_tsv}",
+      "srst2_vibrio_version": "~{srst2_vibrio_version}",
+      "srst2_vibrio_ctxA": "~{srst2_vibrio_ctxA}",
+      "srst2_vibrio_ompW": "~{srst2_vibrio_ompW}",
+      "srst2_vibrio_tcpA_ElTor": "~{srst2_vibrio_tcpA_ElTor}",
+      "srst2_vibrio_toxR": "~{srst2_vibrio_toxR}",
+      "srst2_vibrio_wbeN_O1": "~{srst2_vibrio_wbeN_O1}"
```

Comment:
`srst2_vibrio_tcpA_ElTor` change to `srst2_vibrio_biotype`

and

`srst2_vibrio_wbeN_O1` change to `srst2_vibrio_serotype`